### PR TITLE
VMSS with Linux VMs and Autoscale integration

### PR DESCRIPTION
**Pull request by madhana**
This is a VMSS template for Linux VMs with a loadbalancer and NAT configured to access each VM instance in VMSS. Autoscale is integrated as well.

### CODE REVIEW

File: 201-vmss-ubuntu-autoscale/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+                  "type": "LinuxDiagnostic",
+                  "typeHandlerVersion": "2.1",
+                  "autoUpgradeMinorVersion": false,
+                  "settings": {
+                    "xmlCfg": "[base64(concat(variables('wadcfgxstart'),variables('wadmetricsresourceid'),variables('wadcfgxend')))]",
+                    "storageAccount": "[variables('diagnosticsStorageAccountName')]"
+                  },
+                  "protectedSettings": {
+                    "storageAccountName": "[variables('diagnosticsStorageAccountName')]",
+                    "storageAccountKey": "[listkeys(variables('accountid'), '2015-05-01-preview').key1]",
```
Comment (singhkay) [inline 201-vmss-ubuntu-autoscale/azuredeploy.json]: This api version can probable be changed to 2015-06-15

Comment (madhana) [inline 201-vmss-ubuntu-autoscale/azuredeploy.json]: i made the change Kay, good catch.

**Comment by madhana** [comment]
Kay, i made the changes you suggested, please accept this PR, we want this to be available as soon as possible in main repo, in time for VMSS public preview tomorrow.

**Comment by singhkay** [comment]
Thanks @madhana, merged

**Pull request merged**